packages/crayons-core/src/utils/draggable.ts

```diff
@@ -70,7 +70,20 @@ export class Draggable {
   };
 
   private onDragEnter = (e) => {
-    if (!this.canAcceptDragElement()) {
+    e.preventDefault();
+    e.stopPropagation();
+    if (!this.canAcceptDragElement() || this.isDropNotAllowed()) {
+      // Dispatch a custom event when drag enters the invalid field in container shows error message
+      this.dragContainer.dispatchEvent(
+        new CustomEvent('fwDragEnterBase', {
+          cancelable: true,
+          bubbles: false,
+          detail: {
+            droppedElement: dragElement,
```

Comment:
`dragElement` is set in the above function at onDragStart, are we sure that the same element is correct for this onDragEnter event as well.

Reply:
yeah it is needed for this also